cmd/bootstrap.go

```diff
@@ -256,6 +256,9 @@ func bootstrapCmdRun(cmd *cobra.Command, args []string) {
 		cfg.Tenant,
 		cfg.MfaToken,
 	)
+	if err != nil {
+		zap.S().Fatalf("Unable to fetch keystone token: %s", err.Error())
```

Comment:
```suggestion
		zap.S().Fatalf("Failed to get keystone %s", err.Error())
```
This is consistent with the error message in other files.

Reply:
Done